src/README.md

```diff
@@ -36,7 +36,7 @@ navigate to the folder of your consuming project and run one of the following co
 _published:_
 
 ```
-npm install @tiledb-inc/tiledb-cloud@2.0.2 --save
+npm install @tiledb-inc/tiledb-cloud@2.0.13 --save
```

Comment:
The npm's package version will be 0.0.7 right?

Reply:
This is auto generated inside src.

Reply:
Do we really need this auto generated readme? Should we remove it?

Reply:
tbh i don's see any value having this README i guess we can just remove it, right now it only complicates users.